mopidy_musicbox_webclient/static/index.html

```diff
@@ -350,11 +351,13 @@ <h3 id="h_artistname"></h3>
         <div class="ui-block">
             <form>
                 <p>Search for artists, albums, or specific tracks.
-                <select id="selectSearchService"> <!-- data-native-menu="false"> <!-- multiple="multiple" data-native-menu="false">
+                <select id="selectSearchService"> 
+                    <!-- data-native-menu="false"> 
+                    multiple="multiple" data-native-menu="false">
                     <option data-placeholder="true">Choose services</option> -->
                 </select>
                 <input id="searchinput" placeholder="Search term" class="span2" data-clear-btn="true"
-                       onkeypress="return searchPressed(event.keyCode);" id="appendedInputButton" type="text"/>
```

Comment:
``appendedInputButton`` isn't referenced anywhere that I could find, so I'm hoping that this was just left over after a code refactoring exercise, since long forgotten.